packages/hash/frontend/src/components/layout/PageSidebar/AccountEntityTypeList/EntityTypeItem.tsx

```diff
@@ -0,0 +1,135 @@
+import { Box, BoxProps, styled, Tooltip, Typography } from "@mui/material";
+import { useRef, useState, VFC } from "react";
+import { faEllipsis } from "@fortawesome/free-solid-svg-icons";
+import { usePopupState, bindTrigger } from "material-ui-popup-state/hooks";
+import { Link } from "../../../Link";
+import { FontAwesomeIcon } from "../../../icons";
+import { IconButton } from "../../../IconButton";
+import { EntityTypeMenu } from "./EntityTypeMenu";
+
+type EntityTypeItemProps = {
+  title: string;
+  entityId: string;
+  accountId: string;
+  selected: boolean;
+};
+
+const Container = styled(
+  (
+    props: BoxProps & { hovered: boolean; focused: boolean; selected: boolean },
+  ) => <Box component="li" {...props} />,
+)(({ theme, selected, hovered, focused }) => ({
+  marginLeft: theme.spacing(0.5),
+  marginRight: theme.spacing(0.5),
+  paddingLeft: theme.spacing(3.75),
+  paddingRight: theme.spacing(0.5),
+  borderRadius: "4px",
+  display: "flex",
+  justifyContent: "space-between",
+  alignItems: "center",
+  color: theme.palette.gray[70],
+
+  "&:hover": {
+    backgroundColor: theme.palette.gray[20],
+    color: theme.palette.gray[80],
+  },
+
+  ...((hovered || focused) &&
+    !selected && {
+      backgroundColor: theme.palette.gray[20],
+      color: theme.palette.gray[80],
+    }),
+
+  "&:focus-within": {
+    backgroundColor: theme.palette.gray[20],
+    color: theme.palette.gray[80],
+  },
+
+  ...(selected && {
+    backgroundColor: theme.palette.gray[30],
+    color: theme.palette.gray[80],
+  }),
+}));
+
+export const EntityTypeItem: VFC<EntityTypeItemProps> = ({
+  accountId,
+  entityId,
+  title,
+  selected,
+}) => {
+  const entityMenuTriggerRef = useRef(null);
+  const popupState = usePopupState({
+    variant: "popover",
+    popupId: "entity-menu",
+  });
+  const [hovered, setHovered] = useState(false);
+  const [focused, setFocused] = useState(false);
```

Comment:
Just as for `:hover`, I don't think we should be storing `focused` in the react state unless we cannot use the `:focus` CSS selector to get equivalent functionality. I don't that's the case from reading this file.

Reply:
https://github.com/hashintel/hash/pull/409#discussion_r837648585